sound/soc/soc-generic-dmaengine-pcm.c

```diff
@@ -185,6 +197,12 @@ static struct dma_chan *dmaengine_pcm_compat_request_channel(
 	struct snd_dmaengine_dai_dma_data *dma_data;
 	dma_filter_fn fn = NULL;
 
+	if (rtd->num_cpus > 1) {
+		dev_err(rtd->dev,
+			"%s doesn't support Multi CPU yet\n", __func__);
+		return -EINVAL;
```

Comment:
this function returns a pointer. Should it have been `ERR_PTR(-EINVAL)` or `NULL`?

Reply:
yes, fixed in update (reported independently by kbuild)